packages/contracts-bedrock/src/libraries/SafeCall.sol

```diff
@@ -48,6 +48,27 @@ library SafeCall {
         return _success;
     }
 
+    /// @notice Perform a low level call without copying any returndata and passing all available gas
+    /// @param _target   Address to call
+    /// @param _value    Amount of value to pass to the call
+    /// @param _calldata Calldata to pass to the call
+    function callWithAllGas(address _target, uint256 _value, bytes memory _calldata) internal returns (bool) {
```

Comment:
I don't think we can safely change the solidity version in this file unfortunately, perhaps we just put this function into the contract its used in